src/index.js

```diff
@@ -1,26 +1,33 @@
 'use strict';
 
-var Retriever = require('./retriever');
-var RequestFactory = require('./request-factory');
+var assign = require('lodash.assign');
+var ResultList = require('./result-list');
+
+var _defaults = {
+  page: 1,
+  per_page: 20,
+  safesearch: true,
+  url: 'https://pixabay.com/api'
+};
 
 var pixabayjs = {
+  _auth: {},
+
   authenticate: function(username, key) {
-    this.username = username;
-    this.key = key;
+    this._auth.username = username;
+    this._auth.key = key;
   },
 
-  defaults: function(defaults) {
-    this.defaults = defaults;
+  get defaults () {
+    return _defaults;
+  },
+  set defaults (value) {
+    _defaults = assign(_defaults, value);
```

Comment:
turns out this compounds `_defaults` in a way we really don't want, because assign builds into the first object:
```js
var test = {hey: 1, you: 2};
undefined
var pixa = {
get thing () {
  return test;
},
set thing(val) {
  test = _.assign(test, val);
}
};

pixa.thing // Object {hey: 1, you: 2}
pixa.thing = {me: 5};
pixa.thing // Object {hey: 1, you: 2, me: 5}
pixa.thing = {bro: 'fuuuuuck'};
pixa.thing // Object {hey: 1, you: 2, me: 5, bro: "fuuuuuck"}
```
we also don't want the getter and setter to be working on `_defaults`; we want them to have their own var.  here's the behavior i think we're expecting: 
```js
var other;
var test = {hey: 1, you: 2};
var pixa = {
  get thing () {
    return other;
  },
  set thing(val) {
    other = _.assign({}, test, val);
  }
};

pixa.thing = {};
pixa.thing // Object {hey: 1, you: 2}
pixa.thing = {me: 'you'};
pixa.thing // Object {hey: 1, you: 2, me: "you"}
pixa.thing = {us: 'them'};
pixa.thing // Object {hey: 1, you: 2, us: "them"}
```